JSDemos/Demos/Calendar/Overview/Angular/app/app.component.ts

```diff
@@ -23,8 +23,8 @@ export class AppComponent {
     now: Date = new Date();
     currentValue: Date = new Date();
     firstDay: number = 0;
-    minDateValue: Date = undefined;
-    maxDateValue: Date = undefined;
+    minDateValue: Date = null;
```

Comment:
If we rely on types, it is more logical to see typing as below
```suggestion
    minDateValue: Date | null = null;
```
But our d.ts  has another type list:
```ts
min?: Date | number | string;
```
Won't the compiler break when building the application?
@IlyaKhD do we need to add null at the basic option level?

Reply:
You're right, the declaration should be updated as well:
`min?: Date | number | string | null;`

Reply:
Is it also related to disabledDates?
This is from our d.ts
`disabledDates?: Array<Date> | ((data: { component?: any, date?: Date, view?: string }) => boolean);`